styleguide/source/_patterns/02-molecules/tool.twig

```diff
@@ -0,0 +1,14 @@
+{% set icon, paragraph, type, size, link =
+tool.icon, tool.paragraph, tool.type, tool.size, tool.link
+%}
+
+<a class="ama__tool" href={{ link }}>
+  <icon class="ama__tool--icon {{ icon }}"></icon>
+  <div class="ama__tool--text">
+    {% include '@atoms/paragraph.twig' %}
+    <div class="ama__tool--meta">
```

Comment:
It looks like `--icon`, `--text` and `--meta` are all sub elements of `ama__tool` in which case they should be denoted with a double underline instead of a double hyphen (e.g. `ama__tool__icon`)